compiler-plugin/src/main/kotlin/arrow/meta/internal/kastree/ast/psi/Converter.kt

```diff
@@ -926,13 +926,13 @@ open class Converter {
 internal val PsiElement.ast: Node get() = when(this) {
   is KtClassOrObject -> Converter.convertDecl(this)
   is KtNamedFunction -> Converter.convertFunc(this)
-  is KtExpression -> Converter.convertExpr(this)
+  is KtTypeAlias -> Converter.convertTypeAlias(this)
   is KtWhenCondition -> Converter.convertWhenCond(this)
   is KtWhenEntry -> Converter.convertWhenEntry(this)
   is KtCatchClause -> Converter.convertTryCatch(this)
   is KtImportDirective -> Converter.convertImport(this)
   is KtValueArgument -> Converter.convertValueArg(this)
-  is KtTypeAlias -> Converter.convertTypeAlias(this)
   is KtTypeReference -> Converter.convertTypeRef(this)
+  is KtExpression -> Converter.convertExpr(this)
```

Comment:
TypeAlias can also be considered an Expression, so we're going to want larger fragmented PSI elements to go towards the bottom.